packages/webapp/src/containers/Field/NewField/index.js

```diff
@@ -373,46 +401,23 @@ class NewField extends Component {
                 autoFocus
                 value={this.state.fieldName}
                 onChange={this.handleFieldNameChange}
-                className={'randomClass'}
+                className={styles.buttonContainer}
                 style={{borderColor: '#89D1C7'}}
               />
               }
-            {/* </div> */}
             </FormGroup>
             <FormGroup
               className={styles.centeredForm}>
               <div className={styles.buttonContainer} style={{ bottom: 0 }}>
-                {this.state.isSavePlanDisabled ?
-                    <Button style={{
-                      font: "Open Sans",
-                      fontSize: "16px",
-                      lineHeight: "24px",
-                      borderRadius: "4px",
-                      display: "flex",
-                      alignItems: "center",
-                      textAlign: "center",
-                      letterSpacing: "0.4005px",
-                      backgroundColor: "#D4DAE3",
-                      color: "#282B36",
-                      border: "none",
-                      cursor: "default",
-                      boxShadow: "0px 2px 8px rgba(102, 115, 138, 0.3)",
-                    }} outline>Save Field</Button>
-                    :
-                    <Button style={{
-                      font: "Open Sans",
-                      fontSize: "16px",
-                      lineHeight: "24px",
-                      borderRadius: "4px",
-                      display: "flex",
-                      alignItems: "center",
-                      textAlign: "center",
-                      letterSpacing: "0.4005px",
-                      backgroundColor: "#FCE38D",
-                      color: "#282B36",
-                      border: "none",
-                      boxShadow: "0px 2px 8px rgba(102, 115, 138, 0.3)",
-                    }} outline
+                { 
+                this.state.isSavePlanDisabled ?
+                <Button style={this.getButtonStyles()} outlinonClick={() => {
+                  this.handleModeChange(CREATE_FIELD);
+                  this.setState({ step: this.state.step + 1 });
+                }}>Save Field</Button>
+                :
+                <Button style={this.getActiveButtonStyles()} 
```

Comment:
Doing this in a function will trigger this on every state change, I think it might be better just to handle button styles as a constant, also having two functions for that seems to be a little overkill. just append the active styles to the active button like this
``` style={{...buttonStyles, ..additionalActiveStyles }}```

Reply:
@kikemarquez8 is `..additionalActiveStyles` another constant with he active properties?